app/contributing/plugin-docs.md

```diff
@@ -6,31 +6,30 @@ Plugin documentation is posted on the [Plugin Hub](/hub/).
 All plugin docs, whether developed by Kong or external contributors,
 follow a [specific template](https://github.com/Kong/docs.konghq.com/tree/main/app/_hub/_init/my-extension).
 
-We are currently accepting plugin submissions to our plugin hub from trusted technical partners, on a limited basis. For more information, see the [Kong Partners page](https://konghq.com/partners/).
+We are currently accepting plugin submissions to our plugin hub from trusted technical partners, on a limited basis. 
+For more information, see the [Kong Partners page](https://konghq.com/partners/).
 
 {:.note}
 > The Kong Plugin Hub is a documentation site. We **do not** host plugin source code or downloads.
 
 ## Add a new plugin doc
 
-1. Set up or find your publisher directory:
-  * If you're contributing a plugin developed by Kong, 
- use the existing `_app/_hub/kong-inc` directory.
-  * If you're documenting a plugin created by another company, 
-  create a publisher directory at`_app/_hub/`, such as
- `_app/_hub/company-name`. 
+### Plugins created by Kong Inc
 
-    See other Kong Hub listings for examples of publisher names.
-
-1. Create a subdirectory for your plugin within your publisher directory.
+1. Create a subdirectory for your plugin [within the `_app/_hub/kong-inc` directory](https://github.com/Kong/docs.konghq.com/tree/main/app/_hub/kong-inc/).
 For example, `_app/_hub/kong-inc/your-plugin`.
 
-1. Copy the `/app/_hub/_init/my-extension/_index.md` and 
-the `/app/_hub/_init/my-extension/versions.yml` files into your plugin's subdirectory.
+1. Copy the [`/app/_hub/_init/my-extension/_index.md` and 
+the `/app/_hub/_init/my-extension/versions.yml`](https://github.com/Kong/docs.konghq.com/tree/main/app/_hub/_init/my-extension)
+files into your plugin's subdirectory.
```

Comment:
```suggestion
files into the plugin's subdirectory.
```

Reply:
I don't feel too strongly about your -> the in this doc, but `your` felt odd to me. Feel free to ignore.